stock_picking_invoice_product_group/models/account_invoice.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Alex Comba - Agile Business Group
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, fields
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    related_picking_ids = fields.Many2many(
```

Comment:
Becasuse we cannot modify the type of https://github.com/OCA/stock-logistics-workflow/blob/8.0/stock_picking_invoice_link/stock.py#L60 without breaking the compatibility of those modules which depend on it. Moreover, only with this module has sense to manage this use case. Do you agree?